integration/tests/economy/test_economics.py

```diff
@@ -1201,6 +1350,12 @@ def test_multiple_scheduled_trx(
 
         sol_balance_after = operator.get_solana_balance()
         token_balance_after = operator.get_token_balance(web3_client_sol)
+        operator_inner_balance_after = operator.get_token_balance(web3_client_sol)
+        user_inner_sol_balance_after = web3_client_sol.get_balance(neon_user_with_all_tokens.checksum_address)
+        full_volume_after = operator_inner_balance_after + user_inner_sol_balance_after
+        diff_volume = full_volume_before - full_volume_after
+        assert diff_volume == 0, f"not same, diff={full_volume_before-full_volume_after}"
```

Comment:
why don't you use new methods sum_balances and assert_tokens_volumes_stayed_same?